src/utils/convertPartialDateStringToObj.ts

```diff
@@ -0,0 +1,72 @@
+import { DateObjectSchema } from 'serina.schema';
+import { DATES, PARTIAL_DATES } from 'filters/dates/dates.constants';
+import contains from './contains';
+import { monthStringToNumber } from 'utils';
+import { DateTime } from 'luxon';
+
+function strToInt(dayStr: string, monthStr: string, yearStr: string): DateObjectSchema {
```

Comment:
Might be worth making this a util function at this point

Reply:
Done.